index.js

```diff
@@ -15,4 +15,15 @@ app.post('/notes', (req, res) => {
 
 app.get('/notes', (req, res) => res.json(notes))
 
+app.put('/notes/:id', (req, res) => {
+  const note = notes.find(note => {
+    return note.id === Number(req.params.id)
+  })
+  if (!note) {
+    res.sendStatus(404)
```

Comment:
Don't forget to `return` or use an `else` here or your server will crash. There's nothing preventing the server from trying to respond twice.